pkg/controller/controller_broker.go

```diff
@@ -341,13 +342,13 @@ func (c *controller) reconcileClusterServiceBroker(broker *v1beta1.ClusterServic
 			delete(existingServicePlanMap, payloadServicePlan.Name)
 
 			glog.V(4).Infof(
-				"%s %q: reconciling ClusterServicePlan (K8S: %q ExternalName: %q)",
-				typeCSB, broker.Name, payloadServicePlan.Name, payloadServicePlan.Spec.ExternalName,
+				"ClusterServiceBroker %q: reconciling %s",
```

Comment:
Looks like this may be an unintentional change back to string literal for ClusterServiceBroker

Reply:
fixed here: https://github.com/kubernetes-incubator/service-catalog/pull/1442/files#diff-823813a64f69ecf73ed0f982e7112da0R372